source/MaterialXCore/Interface.cpp

```diff
@@ -317,6 +317,27 @@ InputPtr Input::getInterfaceInput() const
     return nullptr;
 }
 
+void Input::addTokens(StringResolverPtr& resolver) const
```

Comment:
Need to customize inputs so that interfaces are taken into consideration. If there is an interface than use tokens at that level.